### PR TITLE
Not all mpiruns recognize --oversubscribe. OpenMPI does, so check.

### PR DESCRIPTION
**Pull request by hategan**

### CODE REVIEW

File: src/psij/launchers/scripts/mpi_launch.sh [hunk 1]
```diff
@@ -5,10 +5,19 @@ source $(dirname "$0")/launcher_lib.sh
 _PSI_J_PROCESS_COUNT="$1"
 shift
 
+IS_OPENMPI=0
+if mpirun -version | grep "Open MZI" >/dev/null 2>&1; then
```
Comment (andre-merzky) [inline src/psij/launchers/scripts/mpi_launch.sh:9]: There is a typo (`MZI`)

File: src/psij/launchers/scripts/mpi_launch.sh [hunk 2]
```diff
@@ -5,10 +5,19 @@ source $(dirname "$0")/launcher_lib.sh
...
+
 pre_launch
 
 set +e
-mpirun -n $_PSI_J_PROCESS_COUNT "$@" 1>$_PSI_J_STDOUT 2>$_PSI_J_STDERR <$_PSI_J_STDIN
+if "$IS_OPENMPI" == "1"; then
+    mpirun --oversubscribe -n $_PSI_J_PROCESS_COUNT "$@" 1>$_PSI_J_STDOUT 2>$_PSI_J_STDERR <$_PSI_J_STDIN
+else
+    mpirun -n $_PSI_J_PROCESS_COUNT "$@" 1>$_PSI_J_STDOUT 2>$_PSI_J_STDERR <$_PSI_J_STDIN
+fi
```
Comment (andre-merzky) [inline src/psij/launchers/scripts/mpi_launch.sh:20]: Once the number of cases increase (and they will), we should reconsider the code replication.  For now that works.

**Review by andre-merzky — APPROVED** [review]
Marked typo, otherwise LGTM - thanks.

File: src/psij/launchers/scripts/mpi_launch.sh (hunk 1 above)
Comment (hategan) [reply src/psij/launchers/scripts/mpi_launch.sh:9]: Uh, that's not a typo. I was testing both possibilities (string found and string not found) and forgot to revert. Will fix.

**Pull request merged**